### ISSUE TITLE
Accessibility issues on website

### ISSUE THREAD

**Issue by edent** [wpscanteam/wpscan#1532]
As [discussed on Twitter](https://twitter.com/ethicalhack3r/status/1293812418234155012), the contrast on the main website does not meet modern accessibility requirments.
![Screenshot from 2020-08-13 09-28-45](https://user-images.githubusercontent.com/837136/90112247-a4788600-dd47-11ea-8768-1086f72539cc.png)
![Screenshot from 2020-08-13 09-29-23](https://user-images.githubusercontent.com/837136/90112248-a5111c80-dd47-11ea-80ea-bf24c5763068.png)

I'll raise a PR shortly.

**Comment by ethicalhack3r** [comment]
Thanks!
